harold/plugins/deploy.py

```diff
@@ -171,11 +171,7 @@ class DeployHoldListener(DeployListener):
     def _handle_request(self, request):
         reason = request.args['reason'][0]
         salon_name = request.args['salon_name'][0]
-        salon = yield self.monitor.salons.by_name(salon_name)
-        if not salon:
-            request.setResponseCode(400)
```

Comment:
While it would be nice to bubble up the Bad Request error to the client it just adds too many unnecessary complications.